index.html

```diff
@@ -5,17 +5,514 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Coherent UI</title>
-    <link rel="stylesheet" href="styles.css">
+    <link rel="shortcut icon" href="./images/logo.svg" type="image/x-icon">
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+        integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
 </head>
 
 <body>
-    <div class="main-div">
-        <div class="container">
-            <button>V0.0</button>
-            <h1>Coherent UI</h1>
-            <a class="documentation-anchor" href="./documentation.html">Get Started</a>
+    <section class="container">
+        <nav class="sidebar">
+            <div class="sidebar-header">
+                <span> <i class="fab fa-cuttlefish"></i>
+                </span>
+                <span class="title">
+                    <h1>Coherent UI</h1>
+                </span>
+            </div>
+            <div class="sidebar-content">
```

Comment:
#### Make use of more Semantic HTML tags, instead of `div` use `aside` as this is your sidebar.